### ISSUE TITLE
Connected tables do not work anymore

### ISSUE THREAD

**Issue by tomasonjo** [metabase/metabase#11928]
**Describe the bug**
After updating to v0.34.2, connected tables feature has stopped working:

**To Reproduce**
1. Create two tables in PostgreSQL
2. Set the schema to that one table has a foreign key to another
3. Open the table in metabase and click on any primary key of the master table
4. Where you should see the other table under the connected tables feature, there is none

**Expected behavior**
I would expect that the table that is connected with the foreign key shows up under connected tables

**Screenshots**
![empty](https://user-images.githubusercontent.com/19948365/74664828-c4ffc300-519e-11ea-8a7f-afab23177f48.png)

**Severity**
Annoying and will have to downgrade if not solved

**Metabase Diagnostic Info**

```json
{
  "browser-info": {
    "language": "sl-SI",
    "platform": "Linux x86_64",
    "userAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.87 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.5+10",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "11.0.5",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.5+10",
    "os.name": "Linux",
    "os.version": "5.0.0-1025-gcp",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "googleanalytics",
      "postgres",
      "bigquery",
      "h2"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "12.1 (Debian 12.1-1.pgdg100+1)"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.8"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2020-02-04",
      "tag": "v0.34.2",
      "branch": "release-0.34.x",
      "hash": "dab738c"
    },
    "settings": {
      "report-timezone": "Europe/Berlin"
    }
  }
}
```

**Comment by flamber** [comment]
Hi @tomasonjo 
Confirmed on 0.34.2 and `master` too.
I don't have a workaround for this, so right now I would recommend downgrade to 0.34.1.
Remember to always backup before upgrade/downgrade.

Steps to reproduce:
1. Simple question > Sample Dataset > Products
2. Click ID column of a product to see ObjectDetail view
3. The Connected panel is just empty - no errors in console or log.

**Comment by paulrosenzweig** [comment]
Fixed by #11930
